inc/admin/network/class-networksettings.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ * @author  Pressbooks <code@pressbooks.com>
+ * @license GPLv3 (or any later version)
+ */
+// @phpcs:disable Pressbooks.Security.EscapeOutput.OutputNotEscaped
+// @phpcs:disable Pressbooks.Security.NonceVerification.Missing
+// @phpcs:disable Pressbooks.Security.ValidatedSanitizedInput.MissingUnslash
+
+namespace Pressbooks\Admin\Network;
+
+class NetworkSettings {
+
+	const DEFAULT_THEME_OPTION = 'pressbooks_default_book_theme';
+
+	const DEFAULT_THEME = 'pressbooks-malala';
```

Comment:
We will use these constants when adding a hook in pressbooks config plugin to create the default theme option by default when the network is created.

Reply:
Should it be `pressbooks-book` rather than Malala, since the first is open source?

Reply:
yes, you're right. I created a method to get the default theme (see https://github.com/pressbooks/pressbooks/pull/2608/files#diff-ab3df14e8786c533f34c4228942654997df95ceed2493b79d54dcd883945e8b6R102). It basically returns pressbooks-malala is available, otherwise it returns pressbook-book